docs/topics/permissions.md

```diff
@@ -83,6 +83,7 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | SEND_VOICE_MESSAGES                    | `0x0000400000000000` `(1 << 46)` | Allows sending voice messages                                                                                                                                                                                                                              | T, V, S                    |
 | SEND_POLLS                             | `0x0002000000000000` `(1 << 49)` | Allows sending polls                                                                                                                                                                                                                                       | T, V, S                    |
 | USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows user-installed apps to send public responses. When disabled, users will still be allowed to use their apps but the responses will be ephemeral. This only applies to apps not also installed to the server.                                         | T, V, S                    |
+| PIN_MESSAGES                           | `0x0008000000000000` `(1 << 51)` | Allows pinning and unpinning messages                                                                                                                                                                                                                      | T                          |
```

Comment:
```suggestion
| PIN_MESSAGES                           | `0x0008000000000000` `(1 << 51)` | Allows pinning and unpinning messages                                                                                                                                                                                                                      | T, V, S                    |
```

This permission also applies to voice and strange channels cause both have text

Reply:
those dont seem to support pinning

Reply:
Oop. My memory fooled me the. Disregard